### DISCUSSION TITLE
自己部署了rsshub，如何停止日志文件的输出

### DISCUSSION THREAD

**Discussion by cnkmmk** [DIYgod/RSSHub#15480] [Q&A] [answered]
### 这是一个什么样的功能？

自己部署的RSSHUB，想停止日志文件的输出。

### 这个功能可以解决什么问题？

VPS空间比较小，日志把磁盘空间全部占用。

### 这不是重复的功能请求和 RSS 提案

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求及不是 [新的 RSS 提案](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_zh.yml)。

**Comment by TonyRL** [comment, marked as answer]
https://docs.rsshub.app/deploy/config#logging-configurations

**Comment by cnkmmk** [comment]
确实，配置了环境变量之后，输出日志停止了。谢谢
